doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -1,16 +1,43 @@
-[id='environment-authoring-con']
+[id='environment-authoring-con'] 
+ifdef::PAM[]
 = Authoring environment
 You can deploy an environment for creating and modifying processes using {CENTRAL}. It consists of {CENTRAL} for the authoring work and {KIE_SERVER} for test execution of the processes.
+endif::PAM[]
+ifdef::DM[]
+= Authoring or managed server environment
+You can deploy an environment for creating and modifying services using {CENTRAL} and for running them in {KIE_SERVERS} managed by {CENTRAL}. This environment consists of {CENTRAL} and one or more {KIE_SERVERS}.
 
-Depending on your needs, you can deploy either a single authoring environment or a high-availability (HA) authoring environment.
+You can use {CENTRAL} both to develop services and to deploy them to one or several {KIE_SERVERS}. For example, you can deploy test versions of services to one {KIE_SERVER} and production versions to another {KIE_SERVER}.
 
-A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. The {KIE_SERVER} includes an embedded in-memory H2 database engine. This type of environment uses the least possible amount of resources. However, because of the in-memory database, restarting the {KIE_SERVER} pod leads to loss of all process information.
+To avoid accidentally deploying wrong versions to a production {KIE_SERVER}, you can create separate environments to author services (_authoring environment_) and to manage deployment of production services (_managed server environment_). You can use a shared external Maven repository between these environments, so that services developed in the authoring environment are available in the managed server environment. Usually, one {KIE_SERVER} is sufficient for an authoring environment.
 
-An HA authoring environment contains several pods. Both {CENTRAL} and {KIE_SERVER} are provided in scalable pods that can run in parallel and share persistent storage. The database is provided by a separate pod. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are involved in authoring at the same time.
+For {PRODUCT}, the procedures to deploy an authoring environment and a managed server environment are the same. You must first deploy an authoring environment template, consisting of {CENTRAL} and one {KIE_SERVER}. Then, if necessary, you can deploy additional {KIE_SERVER} templates in the same namespace to create a managed server environment with multiple {KIE_SERVERS}.
```

Comment:
I would say that we do not needed to talk about managed environment for RHDM at all. Managed environment is in RHPAM where BC Monitoring can monitor running process and task. In case of RHDM we do not have any process or tasks. I have the feeling that we are more likely talking here about development vs. production environment.

Reply:
"Managed" is simply the term for using BC to deploy services across KIE servers. I don't want to call it production as I don't want to tell the user how many levels of staging they should have.